README.md

```diff
@@ -2,59 +2,56 @@
 
 [![Build Status](https://ci.deuxfleurs.fr/job/TarponSimulator2017/job/master/badge/icon)](https://ci.deuxfleurs.fr/job/TarponSimulator2017/job/master/)
 
-## Download
+## Play it!
 
-[Download Tarpon Simulator 2017 binaries (Windows and Linux)](https://ci.deuxfleurs.fr/job/TarponSimulator2017/job/master/lastSuccessfulBuild/artifact/TarponSimulator2017.zip)
+ 1. Install dependencies
+   * On Linux, you must install SDL2 and mono. On Fedora: `dnf install mono SDL2 SDL2-devel`
+   * On macOS, you must install SDL2 and mono, we don't know how.
```

Comment:
I Guess we could say that tarpon  compatibility with macOS is equivalent to openFrameworks compatibility with Linux, so tarpon is definitely supported on macOS :)
It's the right place to troll isn't it ?